### ISSUE TITLE
Funnel Metadynamics tutorial error

### ISSUE THREAD

**Issue by duerrsimon** [michellab/BioSimSpaceTutorials#11]
This looks like a great repository however I cannot get the funnel metadynamics to run. 

I am using version BioSimSpace 2022.2.1, AmberTools20 and Gromacs 2021.3

I am getting the following error by running the steps in the tutorial both with the pdb file from the repo and with the file generated by the cells above. The generated PDB file contains indeed no box information. 
I also tried using the `solvated` system set up above but this produces a different error (AttributeError: 'Length' object has no attribute 'magnitude')
```
---------------------------------------------------------------------------
IncompatibleError                         Traceback (most recent call last)
Input In [12], in <cell line: 5>()
      2 system = BSS.IO.readMolecules(["input_files/solvated.pdb"])
      4 # Create the funnel parameters.
----> 5 p0, p1 = BSS.Metadynamics.CollectiveVariable.makeFunnel(system)
      7 # Define the collective variable.
      8 funnel_cv = BSS.Metadynamics.CollectiveVariable.Funnel(p0, p1)

File ~/miniconda3/envs/biosimspace/lib/python3.9/site-packages/BioSimSpace/Metadynamics/CollectiveVariable/_funnel.py:751, in makeFunnel(system, protein, ligand, alpha_carbon_name, property_map)
    749 space_prop = property_map.get("space", "space")
    750 if space_prop not in system._sire_object.propertyKeys():
--> 751     raise _IncompatibleError("The system contains no simulation box property!")
    753 # Store the space.
    754 space = system._sire_object.property("space")

IncompatibleError: The system contains no simulation box property!
```

**Comment by duerrsimon** [comment]
This seems resolved in '2022.2.1+38.g4292f322' Perhaps it should be noted with which version the tutorial can be run or a requirements file be added

**Comment by lohedges** [comment]
Thanks for reporting. The tutorials are still a work in progress and will likely need to be adapted in relation to some of the recent tweaks to the code. To use the _exact_ version of BioSimSpace for which they were written, you can pull from Conda using the `workshop` label, i.e.:

```
conda install -c conda-forge -c omnia -c michellab/label/workshop biosimspace
```

However, I wouldn't advise this, since there have been some bugfixes related to the funnel metadynamics work since then.

As you've noted, the funnel collective variable requires a simulation box in order to function, so won't work for a vacuum system, i.e. you need to parameterise and solvate first. I believe that this wasn't originally the case, but needed to be changed since the points needed to be computed in relation to the periodic boundary, e.g. wrapping across the periodic image.

I'll close this for now and add a note to the BioSImSpaceTutorials repo to note that they are a WIP.

Cheers.
